libultraship/libultraship/Lib/ImGui/backends/imgui_impl_opengl3.cpp

```diff
@@ -132,7 +132,10 @@
 #else
 #include <GLES3/gl3.h>          // Use GL ES 3
 #endif
-#elif !defined(IMGUI_IMPL_OPENGL_LOADER_CUSTOM)
+#elif defined(__SWITCH__)
+#define GL_GLEXT_PROTOTYPES 1
+#include "glad/glad.h"
```

Comment:
we're importing a lib here.

```suggestion
#include <glad/glad.h>
```